src/components/inputs/TextField.js

```diff
@@ -587,30 +585,35 @@ function createStyles({placeholderTextColor, centered, multiline}) {
     container: {
     },
     innerContainer: {
+      flexGrow: 1,
       flexDirection: 'row',
-      borderBottomWidth: 1,
-      borderColor: Colors.dark70,
       justifyContent: centered ? 'center' : undefined,
-      flexGrow: 1
+      borderBottomWidth: 1,
+      borderColor: Colors.dark70
     },
     innerContainerWithoutUnderline: {
       borderBottomWidth: 0
     },
     input: {
-      flexGrow: 1,
-      marginBottom: Constants.isIOS ? 10 : 5,
-      padding: 0,
+      flex: 1,
```

Comment:
I checked again regarding the change with `flexGrow`, when I implemented it I've compared it with the behavior of RN's TextInput component, cause I wanted to provide a consistent behavior. 

The following examples should render the same
```
<View row bg-red50>
          <TextField hideUnderline enableErrors={false} _containerStyle={{flex: 1}} placeholder="enter text..." />
          <Text>text</Text>
        </View>
        <View row bg-blue50>
          <TextInput _containerStyle={{flex: 1}} placeholder="enter text..." />
          <Text>text</Text>
        </View>
```

but after the change, they're not. 

if we're still having trouble with supporting rightIcon without changing the `flexGrow` maybe we can to the least, condition it. 
if `rightIcon` passed, use `flex`, otherwise use `flexGrow`

Reply:
The condition won't solve the issue with a View row wrapper...

Reply:
I've added changes and design request for this feature.